config.go

```diff
@@ -174,6 +174,13 @@ func (cfg Config) Build(opts ...Option) (*Logger, error) {
 		return nil, err
 	}
 
+	if cfg.Level == (AtomicLevel{}) {
```

Comment:
The parentheses are unnecessary.

```suggestion
	if cfg.Level == AtomicLevel{} {
```

Reply:
Tried this earlier, but comes with `expected ';', found '{'`